### PR TITLE
[candi] CSI old mount cleaner

### PR DESCRIPTION
**Pull request by zuzzas**
## Description

Cleanup of old CSI mounts that follow pre Kubernetes 1.24 naming convention. Get deployed only on Kubernetes >=1.24 and only if there are any 

Closes https://github.com/deckhouse/deckhouse/issues/5132

## Why do we need it, and what problem does it solve?

It enabled successful Pod volume unmounts when upgrading Kubernetes on the go (without draining a Node).

Closes upstream issue: https://github.com/kubernetes/kubernetes/pull/107065

## What is the expected result?

When a kubelet reports that it cannot delete a Pod because there is a mount (with path adhering to an old format), parse a log statement an `umount` manually. 

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: fix
summary: Create a system unit that manually unmounts pre-1.24 CSI mounts. That should stop stuck Pods while upgrading kubelet without draining the Node.
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @zuzzas. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @zuzzas. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by zuzzas** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.23 5505336462 5505336462-1-con-1-23 /sys/deckhouse-oss/install:pr5153 cloud-user@84.201.156.51

**Comment by name212** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.23 5505336462 5505336462-1-con-1-23 /sys/deckhouse-oss/install:pr5153 cloud-user@84.201.156.5

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @zuzzas. I've started the workflow for label 'e2e/run/yandex-cloud'...

File: candi/bashible/common-steps/node-group/089_manually_unmount_old_csi_mounts_after_kubernetes_1_24.sh.tpl [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Copyright 2021 Flant JSC
```
Comment (RomanenkoDenys) [inline candi/bashible/common-steps/node-group/089_manually_unmount_old_csi_mounts_after_kubernetes_1_24.sh.tpl:1]:
```suggestion
# Copyright 2023 Flant JSC
```

**Comment by z9r5** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.23 5505336462 5505336462-1-con-1-23 /sys/deckhouse-oss/install:pr5153 cloud-user@84.201.156.51

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @zuzzas. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by z9r5** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.25 5529685365 5529685365-1-con-1-25 /sys/deckhouse-oss/install:pr5153 cloud-user@84.201.135.116

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [5180](https://github.com/deckhouse/deckhouse/pull/5180) to the branch [release-1.47](https://github.com/deckhouse/deckhouse/tree/release-1.47) successful!

**Pull request merged**